lib/payment_channel.ts

```diff
@@ -57,6 +57,18 @@ export class PaymentChannel {
       document.contractAddress
     )
   }
+
+  toJSON (): PaymentChannelJSON {
+    return {
+      state: this.state,
+      spent: this.spent,
+      value: this.value,
+      channelId: this.channelId,
+      receiver: this.receiver,
```

Comment:
Hey,

What do you think of using the Serde object below in this toJSON method? The goal of the Serde was to centralize the serialization/deserialization of domain objects like payment channels.

Reply:
That's not needed at all :)